docs/app/pages/components/sections/keyboard/hotkeys/hotkeys.component.html

```diff
@@ -0,0 +1,81 @@
+<div class="hotkey-demo-box aspects-focus-outline" tabindex="0">
+    <div class="hotkey-demo-hint">
+        <div class="hotkey-group-hint">q</div>
+    </div>
+    <div class="hotkey-demo-button-layout">
+        <button class="btn button button-secondary" (keydown)="updateQ($event, 'Active')" (click)="updateQClick('Active')">
```

Comment:
You can specify the key in keydown events to prevent you having to check keycodes in the Typescript code. For simple things like text changes you can just do that in the view for both clicks and keydowns: eg:

```html
<button class="btn button button-secondary" (keydown.space)="qText = 'Active'" (keydown.enter)="qText = 'Active'" (click)="qText = 'Active'" [focusIf]="qSelected === 1">
    <i class="hpe-icon hpe-active"></i>
</button>
```